### PR TITLE
Quick fix.

### PR DESCRIPTION
**Pull request by weerd**
Check it! Is it cool that it returns an empty array()? I'm guess probs not?

### CODE REVIEW

**Comment by blisteringherb** [comment]
:+1:

File: lib/modules/dosomething/dosomething_campaign_group/dosomething_campaign_group.module [hunk 1]
```diff
@@ -122,20 +122,22 @@ function dosomething_campaign_group_get_parent_nid($nid) {
...
-    if ($campaign_field['entity']->status == 1) {
-      $campaigns['published'][] = $campaign;
-    }
-    else {
-      $campaigns['unpublished'][] = $campaign;
+  if (isset($campaign_group_node->field_campaigns)) {
+    foreach ($campaign_group_node->field_campaigns['und'] as $campaign_field) {
+      $campaign = array(
+        'title' => $campaign_field['entity']->title,
+        'call_to_action' => $campaign_field['entity']->field_call_to_action['und'][0]['value'],
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign_group/dosomething_campaign_group.module]: `LANGUAGE_NONE`

**Comment by blisteringherb** [comment]
:+1:

**Comment by weerd** [comment]
huzzah!

**Pull request merged**